### PR TITLE
Add upper chest tracking point

### PR DESCRIPTION
**Pull request by Louka3000**
Eiren gave me 2 extension spine straps, so I'm gonna use them. 
This adds a basic upper chest tracking point and shouldn't affect normal usage (i.e, upperChest is hidden by default, and tracking is the same with just chest but no upperChest). 
This thus allows a bit more precision for people that have the extra trackers.

Needs https://github.com/SlimeVR/SolarXR-Protocol/pull/113

### CODE REVIEW

**Comment by Eirenliel** [comment]
firL

**Review by ImUrX — CHANGES_REQUESTED** [review]
add this on the manual proportions ratio group

**Comment by Louka3000** [comment]
I wrote it down and forgor

**Review by ButterscotchV — APPROVED** [review]

File: server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ private void initializeComputedHumanPoseTracker() {
 					VRServer.getNextLocalTrackerId(),
 					"human://CHEST",
 					"Computed chest",
-					TrackerPosition.CHEST,
+					TrackerPosition.UPPER_CHEST,
```
Comment (ImUrX) [inline server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java:147]: is this fine?

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java (hunk 1 above)
Comment (Louka3000) [reply server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java:147]: It really doesn't matter if it's CHEST or UPPER_CHEST as long as it's changed in the rest of the code (doesn't affect config or solarxr I mean). 
So yes it's fine, I did change it in other parts of the code already.

**Review by ImUrX — APPROVED** [review]

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**